cg/cli/upload/delivery_report.py

```diff
@@ -32,8 +30,12 @@
 def delivery_reports(context, print_console, force_report):
     """Generate delivery reports for all cases that need one"""
 
+    if not context.obj.get("analysis_api"):
+        context.obj["analysis_api"] = MipDNAAnalysisAPI(context.obj)
+    analysis_api = context.obj["analysis_api"]
+
```

Comment:
Could this be done with
```suggestion
    analysis_api = context.obj.get("analysis_api", MipDNAAnalysisAPI(context.obj))
```